lms/djangoapps/teams/teams_features.py

```diff
@@ -1,122 +1,25 @@
-import logging
-
-from django.conf import settings
-from web_fragments.fragment import Fragment
-
+"""Methods for teams modifications"""
 from courseware.courses import get_course_by_id
-from courseware.model_data import FieldDataCache
-from courseware.module_render import get_module_for_descriptor
-from openedx.core.djangoapps.crawlers.models import CrawlersConfig
 from student.models import CourseEnrollmentManager
 from opaque_keys.edx.locator import BlockUsageLocator
-
-from xmodule.x_module import STUDENT_VIEW
+from opaque_keys import InvalidKeyError
 
 CONTENT_DEPTH = 2
-SECTION_NAME = getattr(settings, "TEAM_SECTION", "Teams")
-LOG = logging.getLogger(__name__)
-
-
-class ModifyTeams(object):
-    """this class alows to modifies the team's behavior"""
 
-    def __init__(self, request, user, course_key):
-
-        self.effective_user = user
-        self.course_key = course_key
-        self.course = get_course_by_id(self.course_key, CONTENT_DEPTH)
-        self._prefetch_and_bind_course(request)
-
-        self.section = self._find_section()
-        self.subsection = self._find_subsection()
-
-    def get_fragment(self):
-        """This method allows to get a fragment from a course's section"""
-        unit = self._find_unit()
-        if unit:
-            frag = unit.render(STUDENT_VIEW)
-            return frag
+def get_replacement_location_id(course_key):
+    """
```

Comment:
please complement the docstring. Here we are extracting a very special location id corresponding to a configuration that comes from the teams configuration of the course. I think it's important to mention that